modules/test/playwright/fixtures/objectPagesTest.ts

```diff
@@ -5,13 +5,18 @@
 
 import {test} from '@playwright/test';
 
+import {AddObjectDefinitionModalPage} from '../pages/object-web/AddObjectDefinitionModalPage';
```

Comment:
Hi @danvb112, what do you think about using the same filename for pages and tsx elements. I know that `ModalAddObjectDefinitionsPage` might not be the best choice. We can bring this conversation to our gathering front end to define the common names for modal files. What do you think?

Reply:
Initially I thought about two things.

1. Keep the page with the name of the component and then change the name of the modals and therefore the pages.

2. Rename the modals now and keep the page name like this.

However, I believe that the second option would not be valid for this moment since we will be adding unnecessary complexities to our current sprint. What do you think?

Reply:
Thanks @igor-franca, I agree that we could let the page names with the same name of our React components will be more easier to us navigate the code this way